routes/web.php

```diff
@@ -36,6 +36,9 @@
         Route::get('/events',[AdminController::class,'showEvents'])->name('showEvents');
         Route::get('/events/create',[AdminController::class,'createEvent'])->name('createEvent');
         Route::post('/events/store',[AdminController::class,'storeEvent'])->name('storeEvent');
+        Route::get('/events/edit/{event}',[AdminController::class,'editEvent'])->name('editEvent');
+        Route::patch('/events/update/{id}',[AdminController::class,'updateEvent'])->name('updateEvent');
+        Route::delete('/events/destroy/{id}',[AdminController::class,'destroyEvent'])->name('destroyEvent');
```

Comment:
please apply the same approve with my comment above

Reply:
Is it OK that the same URI as update?

Reply:
yes it's okay since they don't have the same request method, for the update, it's using `patch` and the `destroy` is using `delete`